### PR TITLE
Update to support dual-stack IP URL.

### PR DESCRIPTION
**Pull request by kcorey**
@aldevet @arj154 Could you guys take a look?

### CODE REVIEW

**Comment by arj154** [comment]
will 'AFAmazonS3USStandardRegion' be used elsewhere in the code? LGTM otherwise.

**Comment by kcorey** [comment]
Nope, we just go with the default server.  We don't, by rights, try to do anything specific about getting the server closest to us geographically.

**Comment by arj154** [comment]
Ah ok, then LGTM

File: AFAmazonS3Client/AFAmazonS3Client.m [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 static NSString * AFAmazonS3BaseURLStringWithBucketInRegion(NSString *bucket, NSString *region, BOOL useSSL) {
     if (!region) {
-        region = AFAmazonS3USStandardRegion;
+        region = AFAmazonS3ClientDefaultBaseURLString;
```
Comment (kcorey) [inline AFAmazonS3Client/AFAmazonS3Client.m]: Yep, we've always defaulted to the US one, for some reason, prior to this.  However, according to Amazon's documentation, that shouldn't matter as it virtualises your request based on your location.

Comment (aldevet) [inline AFAmazonS3Client/AFAmazonS3Client.m]: Ok, it looks good then 👍

**Pull request merged**